app/Commands/LocalDocker/Xdebug.php

```diff
@@ -0,0 +1,143 @@
+<?php declare( strict_types=1 );
+
+namespace App\Commands\LocalDocker;
+
+use App\Commands\DockerCompose;
+use App\Services\Docker\Local\Config;
+use Illuminate\Support\Facades\Artisan;
+
+/**
+ * Enable/Disable Xdebug in the php-fpm container.
+ *
+ * @package App\Commands\LocalDocker
+ */
+class Xdebug extends BaseLocalDocker {
+
+    /**
+     * The path to the xdebug.ini in the container.
+     */
+    public const XDEBUG_CONFIG_PATH = '/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini';
+
+    /**
+     * The signature of the command.
+     *
+     * @var string
+     */
+    protected $signature = 'xdebug {action? : on|off}';
+
+    /**
+     * The description of the command.
+     *
+     * @var string
+     */
+    protected $description = 'Enable/disable Xdebug in the php-fpm container to increase performance on MacOS';
+
+    /**
+     * Execute the console command.
+     *
+     * @param  \App\Services\Docker\Local\Config  $config
+     *
+     * @return int
+     */
+    public function handle( Config $config ) {
+        $action = $this->argument( 'action' );
+
+        chdir( $config->getDockerDir() );
+
+        if ( empty( $action ) ) {
+            $result = Artisan::call( DockerCompose::class, [
+                '--project-name',
+                $config->getProjectName(),
+                'exec',
+                '--user',
+                'root',
+                'php-fpm',
+                'bash',
+                '-c',
+                sprintf( '[[ -f %s ]]', self::XDEBUG_CONFIG_PATH ),
+            ] );
+
+            if ( self::EXIT_SUCCESS === $result ) {
+                $this->info( 'xdebug is on' );
+            } else {
+                $this->info( 'xdebug is off' );
+            }
+
+            return self::EXIT_SUCCESS;
+        }
+
+        if ( 'on' === $action ) {
+            $this->enable( $config );
+            $this->reload( $config );
+            $this->info( 'xdebug enabled' );
+        } elseif ( 'off' === $action ) {
+            $this->disable( $config );
+            $this->reload( $config );
+            $this->info( 'xdebug disabled' );
+        } else {
+            $this->error( sprintf( 'Invalid argument: %s. Allowed values: on|off', $action ) );
+        }
+
+        return self::EXIT_SUCCESS;
+    }
+
+    /**
+     * Enable xdebug by renaming the .ini file.
+     *
+     * @param  \App\Services\Docker\Local\Config  $config
+     */
+    protected function enable( Config $config ): void {
+        Artisan::call( DockerCompose::class, [
```

Comment:
Should this check to see if it is already enabled/disabled before attempting to change it? I presume it just gives a file not found error and doesn't do anything, so that's probably sufficient.

Reply:
I hide the output of the `mv` commands, so it just fails and reloads without an issue and without showing the user anything.